### PR TITLE
Links markers  list dialogs as checklist

### LINKED ISSUE #239: Links window : Add the ". . ." to comments and assignments columns as ion checklist
Link window opened from anchor
image

This UI isn't clear enough, we need the ". . ." label as in checklist to show user that it's editable (if user as write permissions)

Checklist with the ". . ." label :
image

### PR DESCRIPTION
**Pull request by le-jeu**
Link list and marker list dialog as checklist dialogs
I'm also considering having a `.getItems` function like `.getFields`

Need some audit on UI update.
We need to fix current update issues before (see #238)

Fix #239 #240 #242

### CODE REVIEW

File: src/code/dialogs/checklist.js [hunk 1]
```diff
@@ -47,7 +44,13 @@ const OperationChecklistDialog = WDialog.extend({
 
   _displayDialog: async function () {
     const operation = getSelectedOperation();
-    this.sortable = this.getListDialogContent(operation, 0, false); // defaults to sorting by op order
+    loadFaked(operation);
```
Comment (cloudkucooland) [inline src/code/dialogs/checklist.js:47]: isn't this too "automatic" for you?

File: src/code/dialogs/checklist.js [hunk 2]
```diff
@@ -268,9 +268,14 @@ const OperationChecklistDialog = WDialog.extend({
         },
       },
     ];
+  },
+
+  getListDialogContent: function (operation, items, sortBy, sortAsc) {
```
Comment (cloudkucooland) [inline src/code/dialogs/checklist.js:273]: elegant solution. I like it.

**Review by cloudkucooland — APPROVED** [review]
I like it. Much cleaner than what my initial thought was.

File: src/code/dialogs/checklist.js (hunk 1 above)
Comment (le-jeu) [reply src/code/dialogs/checklist.js:47]: it is, this feature is controlled by the user settings (disabled by default)

Comment (cloudkucooland) [reply src/code/dialogs/checklist.js:47]: ah, ok

**Pull request merged**